portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+/** Comment data class */
+public class Comment {
+
+  /* Comment text */
+  private String commentText;
+
+  /* Comment sentiment */
+  private boolean sentimentScoreFlag;
+
+  private float sentimentScore;
+
+  /**
+   * Constructs a comment object with given text
+   *
+   * @param commentText the comment text
+   */
```

Comment:
When will a `Comment` ever not have a sentiment score? 
Can we get rid of the flag and add the score to this constructor so we can never init a comment without it.

Reply:
A sentiment score, in my opinion, is not a *required* piece of information to store a comment. Thus, I believe it is reasonable for it to be stored as an optional field.

Reply:
Why is it not required? The code as written will always supply a sentiment score right? No point in making something more generic than is useful, because YAGNI

Reply:
Good point. I will make the sentiment score required in this PR.